ports/qemu/README.md

```diff
@@ -110,3 +122,5 @@ The following options can be specified on the `make` command line:
 - `QEMU_DEBUG_ARGS`: defaults to `-s` (gdb on TCP port 1234), but can be overridden
   with different qemu gdb arguments.
 - `QEMU_DEBUG_EXTRA`: extra options to pass to qemu when `QEMU_DEBUG=1` is used.
+- `RUN_NATMODTESTS_ARGS`: pass in extra flags for `run-natmodtests.py` when invoked
+  via `make test_natmod`.
```

Comment:
I'm not sure if `RUN_NATMODTESTS_ARGS` should be exposed to the user as a public-facing variable.  It's set by the board to configure things like the arch.

If we want to give control to the user, it should probably be via a `RUN_NATMODTESTS_EXTRA` variable.  And maybe that should default to `extmod/{deflate,framebuf,heapq,random_basic,re}*.py` for the `VIRT_RV32` board (with the idea to add more tests for ARM-based boards in the future).

Reply:
Valid point.  I opted to remove the entry from the documentation instead of adding another user-facing variable as that, if needed, can be done when enabling natmod testing on Arm.